python/lsst/the/monster/assemble.py

```diff
@@ -299,8 +300,8 @@ def run(self,
                 if colorterm_spline.source_color_field_1 == 'psfMag_g_flux':
```

Comment:
Related to my comment on the file, is this necessary?  And maybe the fixes below are what make things work for the first pass as well as the subsequent passes.